.github/workflows/ersem.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: build-ersem
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build-fabm-ersem:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout branch
+          uses: actions/checkout@v2
+        - name: Setup python
+          uses: actions/setup-python@v2
+          with:
+              python-version: '3.x'
+        - name: Installing wheel and numpy
+          run: python -m pip install wheel numpy
+        - name: Cloning FABM
+          run: git clone https://github.com/fabm-model/fabm.git
+        - name: Building FABM-ERSEM
+          run: |
+              mkdir build && cd build
+              cmake ../fabm/src/drivers/python -DFABM_ERSEM_BASE=..
+              make install
+        - name: Check installation
+          run: python -c "import fabm"
+  build-gotm-fabm-ersem:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout branch
+          uses: actions/checkout@v2
+        - name: Setup python
```

Comment:
I think this job can do without python?

Reply:
Ahh yes, the `gotm-fabm-ersem` does not need `python` I'm guessing?

Reply:
Do you think the same would be try for `gfortran`? It was using `f95` before

Reply:
yes gotm-fabm-ersem does not require python. On the systems I'm familiar with, f95 is an alias for gfortran. If so, that would be fine as long as the version is 4.9 or higher. Did cmake report more details about the compiler (e.g. version)?

Reply:
I removed `gfortran` and it seemed to be fine. You can see the `cmake` report but not the full log file. I was hoping that there would be a debug mode (similar to the one in travis) which one can use to debug the yaml file, I have not found this yet...